ugvc/mrd/ppmSeq_utils.py

```diff
@@ -950,7 +953,7 @@ def read_dumbell_leftover_from_trimmer_histogram(trimmer_histogram_extra_csv, le
         dataframe with dumbell leftover stats
     """
     df_dumbbell_leftover = pd.read_csv(trimmer_histogram_extra_csv)
-    expected_columns = ["Dumbbell_leftover_start_match", "count"]
+    expected_columns = ["dumbbell_leftover_start_match", "count"]
```

Comment:
consider moving the string "dumbbell_leftover_start_match" to a const or enum if you use it in multiple places